lib/round.rb

```diff
@@ -0,0 +1,49 @@
+class Round
+  attr_reader :deck, :turns, :index, :number_correct, :correct_array, :category_array
+  attr_accessor :card_deck
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+    @index = 0
+    @correct_array = []
+    @category_array = []
+    @card_deck = deck.cards
+  end
+
+  def current_card
+    @card_deck[0]
+  end
+
+  def take_turn(guess)
+    another_turn = Turn.new(guess, current_card)
+    @category_array << another_turn.card.category
```

Comment:
you do already have a method for cards_in_category that give you back the cards in a particular category. For that reason, I don't think it's necessary that you create this variable. Similarly, on line 43, when you call to this variable, you could just call to the method that returns the cards in a particular category and count them up.